.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libpwq"]
+	path = libpwq
+	url = https://github.com/dgrove-oss/libpwq.git
```

Comment:
When I did the integration with swift/utils/build-script last week I found some autotools fixes (similar issues as #34 in libdispatch) that need to get upstreamed to libpwq.  I have a unmerged pull request open (https://github.com/mheily/libpwq/pull/8) but in the short run I think we should go with my fork until everything is working & integrated into the swift build (there may be more fixes to libpwq that we don't know about yet).